hammer_cli_foreman_ansible.gemspec

```diff
@@ -18,6 +18,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = Dir['{test}/**/*']
 
   spec.add_dependency 'hammer_cli_foreman', '>= 0.12.0'
+  spec.add_dependency 'hammer_cli_foreman_remote_execution', '~> 0.2.2'
```

Comment:
I guess it should be unpinned since it can lead to version conflicts.

Reply:
@ofedoren should I leave or remove this?

Reply:
I'd personally rather removed it.

Reply:
Thanks for the clarification!